tutorials/nodes/run-sintrop-with-docker.md

```diff
@@ -1 +1,72 @@
-## Run Sintrop with docker
\ No newline at end of file
+## Run Sintrop with docker
+
+## Run Sequoia with docker
+
+## Pre requisites
+- Docker installed
+
+
+## Run Project
+
+### Build sin_geth
+```
+docker build -t sin_geth .
+# Build without cache
+docker build --no-cache -t sin_geth .
+```
+
+### Run sin_geth
+```
+docker run -p=30303:30303 -p=8545:8545 -it -v /home/user/sequoia_volume:/go-sintrop/sequoia_node  sin_geth
```

Comment:
The directory is not sequoia_volume, change to sintrop_node

Reply:
fixed